### DISCUSSION TITLE
Full screen low latency invalid

### DISCUSSION THREAD

**Discussion by nana123126** [ppy/osu#23542] [Q&A] [answered]
### Type

Performance

### Bug description

It seems that when osulazer is running in virtual memory, the full screen low latency will not work.

Sometimes I notice that turning off the osulazer frees up a lot of physical memory, and at that time the game has low latency (while at high latency, the opposite is true). Most of the time, the computer does not provide physical memory to the osulazer, which causes the game to be too laggy and have a high delay. I do not know which background programs to close, but I know that these background programs will not have too big a problem even if they are run in virtual memory, so I hope that the osulazer can try to use physical memory instead of virtual memory when running, and the osulazer can give options in the settings, this will satisfy all players. (Machine translation)

Oh, that doesn't seem to be the case. Later, I found that it wasn't virtual memory. Now, I can't find the reason. But I found that if the delay is low, there will be picture tearing, and if the delay is high, there will be no picture tearing (in full screen mode). It works just like window mode.

### Version

2023.511.0-osulazer

### Logs

[updater.log](https://github.com/ppy/osu/files/11470907/updater.log)
[performance.log](https://github.com/ppy/osu/files/11470908/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11470909/runtime.log)

**Comment by peppy** [comment]
Sounds like you may be running on a very low spec PC. Can you provide PC specs?

**Reply by nana123126** [reply]
CPU: Inter(R) Core(TM) i5-2550K CPU @ 3.40GHz,
RAM: 8.0 GB DDR3,
Disk: Samsung SSD 860 EVO 250GB , WDC WD5000AZLX-00JKKA0 , ( osulazer in WDC WD5000AZLX-00JKKA0 ,C: )
GPU: NVIDIA GeForce GTX 660,
System type: 64 bit operating system, x64 based processor,

**Reply by peppy** [reply]
Virtual memory should not be being used with that setup.

**Reply by nana123126** [reply]
> Virtual memory should not be being used with that setup.

However, I recently found that it does not seem to have much to do with virtual memory. I can achieve the effect of low latency by turning down the resolution and then restoring the resolution to the default. (When I turn down the resolution, it is already low latency. It is not about fps, because turning down the resolution does not increase fps. In this case, even if I move the osulazer to virtual memory, it will not cause delay.)

**Reply by nana123126** [reply, marked as answer]
Okay, all of this is useless. I've been struggling with NVIDIA settings and various other things for so long (although the latency has decreased in window mode), in fact, all I need to do is close the top window
(I have software on my computer that forces the window to be at the front and hides, which caused me not to notice this problem)。
